app/models/manageiq/providers/ibm_power_hmc/infra_manager/host.rb

```diff
@@ -1,6 +1,46 @@
 class ManageIQ::Providers::IbmPowerHmc::InfraManager::Host < ::Host
   include ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCaptureMixin
 
+  def validate_stop
+    {:available => true, :message => nil}
+  end
+
+  def validate_shutdown
+    {:available => true, :message => nil}
+  end
+
+  def validate_start
+    {:available => true, :message => nil}
+  end
```

Comment:
@CallisteH you probably want to check the current power state of the host (e.g. you can't start a host that is already on)

Reply:
Hi Adam, is this a good way to do it like this?
For example : 
```
  def validate_shutdown
    {:available => self.vms_off, :message => nil}
  end
```

where `vms_off` is the boolean function that returns true if all vms are shutdown

Reply:
@CallisteH You'd want to set a specific message too (like "Cannot shutdown Host while some VMs are still running").

Reply:
@CallisteH For your vm_offs method, you cannot perform REST requests to the HMC here! You need to look at the VM states in the inventory instead.